### PR TITLE
[WorldMoodTracker] Add scale to map

### LINKED ISSUE #216: [WorldMoodTracker] Add a scale for the intensity of mood
- Issue type: Feature request

### Short Description

The desired result is a scale on the map which would be helpful for users in visualising the different colours on it.

home-1

### Would you like to work on the issue?
Yes.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short Description
Fixes #216.

Test link: http://odyssy.singhpratyush.in:8080/WorldMoodTracker/.

**Changes:**
* Added scale to the UI
* Added `axios` as dependency.
* Corrected color on map over.
* Inverted gradient color direction according to proposal.

**Screenshots for the change:**
screenshot from 2017-06-26 03 36 26

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@singhpratyush test link?

**Comment by singhpratyush** [comment]
@kavithaenair: Added in PR description.

**Comment by mariobehling** [comment]
@singhpratyush Please actively request reviews from your co-developers using the "@" feature and ping them on chat to get your review.

**Comment by singhpratyush** [comment]
@Achint08 @djmgit @hemantjadon: Please review.

File: WorldMoodTracker/js/components/ScaleBar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export default class ScaleBar extends React.Component {
+    render() {
+        return (
+            <div className={[bootstrapStyles.container, customStyles.scaleBarContainer].join(' ')}>
+                <div className={[bootstrapStyles.textCenter, customStyles.scaleBarText].join(' ')}>
+                    <span className={bootstrapStyles.pullLeft}>Negative</span>
+                    <span>Moderate</span>
+                    <span className={bootstrapStyles.pullRight}>Positive</span>
+                </div>
+
```
Comment (djmgit) [inline WorldMoodTracker/js/components/ScaleBar.js]: Any reason for this empty line?

File: WorldMoodTracker/js/components/ScaleBar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                <div className={[customStyles.scaleBar].join(' ')}>
+
```
Comment (djmgit) [inline WorldMoodTracker/js/components/ScaleBar.js]: here too

**Comment by singhpratyush** [comment]
@djmgit: Updated.

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**